src/vs/workbench/contrib/searchEditor/browser/searchEditor.contribution.ts

```diff
@@ -207,6 +207,10 @@ registry.registerWorkbenchAction(
 registry.registerWorkbenchAction(SyncActionDescriptor.create(RerunSearchEditorSearchAction, RerunSearchEditorSearchAction.ID, RerunSearchEditorSearchAction.LABEL,
 	{ mac: { primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KEY_R } }, ContextKeyExpr.and(SearchEditorConstants.InSearchEditor)),
 	'Search Editor: Search Again', category);
+
+registry.registerWorkbenchAction(SyncActionDescriptor.create(FocusQueryEditorWidgetAction, FocusQueryEditorWidgetAction.ID, FocusQueryEditorWidgetAction.LABEL,
+	{ mac: { primary: KeyCode.Escape } }, ContextKeyExpr.and(SearchEditorConstants.InSearchEditor)),
```

Comment:
This can just be `{ primary: KeyCode.Escape }`, rather than being `mac` specific. Some of the other bindings are macOS specific because Windows binds them to system commands, but that isn't the case here.

Reply:
🙈
Fixed https://github.com/microsoft/vscode/pull/94799/commits/fb4b6284aa20c5c5b086632d8081993d5f90db43